lib/curator.rb

```diff
@@ -0,0 +1,77 @@
+require './lib/artist'
+require './lib/photograph'
+require './lib/fileio'
+
+class Curator
+  attr_reader :photographs, :artists
+  def initialize
+    @photographs = []
+    @artists = []
+  end
+
+  def add_photograph(photgraph)
+    @photographs.push photgraph
+  end
+
+  def add_artist(artist)
+    @artists.push artist
+  end
+
+  def find_artist_by_id(id)
+    artist_by_id = nil
+    @artists.each do |artist|
+      if artist.id == id
+        artist_by_id = artist
+      end
+    end
+    artist_by_id
```

Comment:
hint: if you do an if inside a .each you could probably look at find or find_all to collect what you're looking for!